lib/debug/session.rb

```diff
@@ -2524,6 +2524,16 @@ def daemon(*args)
 
   module TrapInterceptor
     def trap sig, *command, &command_proc
+      sym =
+        case sig
+        when String
+          sig.to_sym
+        when Integer
+          Signal.signame(sig)&.to_sym
+        else
+          sig
+        end
+
```

Comment:
@ko1 Thank you for your fast check https://github.com/ruby/debug/pull/976.

And your code is very well cared for and excellent.

I think the following code will be more better if the fix is applied?
https://github.com/ruby/debug/pull/977/files#diff-c87adef8ace662a03249d206e6812162256bbcecabe18cd51861d1e1cd9cdfcaR2537

`case sig&.to_s&.to_sym` → `case sym`

How do you think?